### PR TITLE
{tools}[foss/2023a] gcsfs v2023.12.2.post1

### PR DESCRIPTION
**Pull request by pavelToman**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/g/gcsfs/gcsfs-2023.12.2-foss-2023a.post1.eb [hunk 1]
```diff
@@ -0,0 +1,91 @@
+easyblock = "PythonBundle"
+
+name = 'gcsfs'
+version = '2023.12.2'
+versionsuffix = '.post1'
```
Comment (boegel) [inline easybuild/easyconfigs/g/gcsfs/gcsfs-2023.12.2-foss-2023a.post1.eb:5]: @pavelToman the `.post1` is just part of the version, so:

```suggestion
version = '2023.12.2.post1'
```

Do note that this implies renaming the easyconfig file as well

File: easybuild/easyconfigs/g/gcsfs/gcsfs-2023.12.2-foss-2023a.post1.eb [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+homepage = 'https://github.com/fsspec/gcsfs'
+description = """Pythonic file-system interface for Google Cloud Storage."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+# builddependencies = [('protobuf', '24.0')]
```
Comment (boegel) [inline easybuild/easyconfigs/g/gcsfs/gcsfs-2023.12.2-foss-2023a.post1.eb:12]: can be removed?

File: easybuild/easyconfigs/g/gcsfs/gcsfs-2023.12.2-foss-2023a.post1.eb [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    }),
+    ('fsspec', version, {
+        'checksums': ['8548d39e8810b59c38014934f6b31e57f40c1b20f911f4cc2b85389c7e9bf0cb'],
+    }),
+    ('google-auth', '2.26.2', {
+        'modulename': 'google.auth',
+        'checksums': ['97327dbbf58cccb58fc5a1712bba403ae76668e64814eb30f7316f7e27126b81'],
+    }),
+    ('google-auth-oauthlib', '1.2.0', {
+        # 'modulename': 'google.auth.oauthlib',
```
Comment (boegel) [inline easybuild/easyconfigs/g/gcsfs/gcsfs-2023.12.2-foss-2023a.post1.eb:75]: can be removed?

File: easybuild/easyconfigs/g/gcsfs/gcsfs-2023.12.2-foss-2023a.post1.eb [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    }),
+    ('google-cloud-storage', '2.14.0', {
+        'modulename': 'google.cloud.storage',
+        'checksums': ['2d23fcf59b55e7b45336729c148bb1c464468c69d5efbaee30f7201dd90eb97e'],
+    }),
+    ('requests', '2.31.0', {
+        'checksums': ['942c5a758f98d790eaed1a29cb6eefc7ffb0d1cf7af05c3d2791656dbd6ad1e1'],
+    }),
+    (name, version, {
+        'source_tmpl': '%(name)s-%(version)s%(versionsuffix)s.tar.gz',
```
Comment (boegel) [inline easybuild/easyconfigs/g/gcsfs/gcsfs-2023.12.2-foss-2023a.post1.eb:86]: this can be removed if you integrate `.post1` in `version`

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ jsc-zen2

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3171.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/5ef6ff3e7346d4021219e7fe2645ecec for a full test report.

**Comment by boegel** [comment]
Going in, thanks @pavelToman!

**Pull request merged**